src/styles/_mixins.scss

```diff
@@ -112,6 +112,18 @@
     }
 }
 
+@mixin screenMobile {
```

Comment:
I agree, and it would be nice to discuss how we should implement different screen sizes. They should maybe be called sm, lg instead of mobile as I have written.